src/web/components/CreateSessionForm.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useContext } from 'react';
+import useStyles from 'react-with-styles/lib/hooks/useStyles';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import { ColorResult, SketchPicker } from 'react-color';
+import createStylesFn from ':web/theme/createStylesFn';
+import StorageUtil from ':web/utils/storageUtil';
+import {
+  FIBONACCI,
+  T_SHIRT,
+  DEFAULT_TIME_LIMIT,
+  DEFAULT_COLOR,
+} from ':web/constants';
+import { ThemeContext } from ':web/theme/SessionThemeProvider';
+import { createThemeWithPrimaryColor } from ':web/theme/DefaultTheme';
+
+const stylesFn = createStylesFn(({ unit }) => ({
```

Comment:
not blocking, but all the styling in this app will eventually be ported to use Linaria (which uses styled components, rather than stylesheets)

See [here](https://github.com/Xapphire13/planning-poker/blob/fb3b137077681efd3d74aa2fe9e9e1b968d3f2c1/src/web/components/SessionParticipants.tsx#L13-L19) for an example.

Reply:
Styling react components is definitely my biggest frustration, so I think I'll leave this for another PR 😆